### PR TITLE
Colormap Renderer Sample

### PR DESCRIPTION
**Pull request by ghost**
Demonstrates applying a color map to a grayscale raster layer.

### CODE REVIEW

**Comment by ghost** [comment]
@john0005 Could you also review this one?

File: src/main/java/com/esri/samples/raster/colormap_renderer/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<h1>Colormap Renderer</h1>
+
+<p>How to use a colormap renderer on raster layer. Colormap renderers can be used to replace values of a raster with
```
Comment (john0005) [inline src/main/java/com/esri/samples/raster/colormap_renderer/README.md]: on a raster layer

File: src/main/java/com/esri/samples/raster/colormap_renderer/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<h1>Colormap Renderer</h1>
+
+<p>How to use a colormap renderer on raster layer. Colormap renderers can be used to replace values of a raster with 
+a color based on the original value brightness.</p>
+
+<p><img src="ColormapRenderer.png"/></p>
+
+<h2>How it works</h2>
+
+<p>To apply a `ColormapRenderer` to a `RasterLayer`:</p>
```
Comment (john0005) [inline src/main/java/com/esri/samples/raster/colormap_renderer/README.md]: Replace \`ColormapRenderer\` with  \<code>ColormapRenderer\</code>

File: src/main/java/com/esri/samples/raster/colormap_renderer/ColormapRendererSample.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+          .collect(Collectors.toList());
+
+      // create a colormap renderer
+      ColormapRenderer colormapRenderer = new ColormapRenderer(gradient);
+
+      // set the colormap renderer on the raster layer
+      rasterLayer.setRasterRenderer(colormapRenderer);
+
+      // set viewpoint on the raster
+      rasterLayer.addDoneLoadingListener(() -> mapView.setViewpointGeometryAsync(rasterLayer.getFullExtent()));
```
Comment (john0005) [inline src/main/java/com/esri/samples/raster/colormap_renderer/ColormapRendererSample.java]: Like it better when slightly zoomed out, `mapView.setViewpointGeometryAsync(rasterLayer.getFullExtent(), 150)`

**Review by john0005 — COMMENTED** [review]
Looks good minor changes

**Comment by ghost** [comment]
@john0005 Good catch. Thanks. I've addressed these so I'll go ahead and merge now.

**Pull request merged**